happy/conf/main_config.json

```diff
@@ -1,10 +1,12 @@
 {
     "configuration_file":"~/.happy_conf.json",
     "log_directory":"/tmp",
+    "happy_log_environ":"HAPPY_LOG_DIR",
```

Comment:
Is there a strong value in adding this additional level of indirection to determine the name of the environment variable to read for the Happy log directory?